bin/tests/bootstrap-unit.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/local/bin/bats
+
+lib_dir="/usr/local/lib"
+
+load "$lib_dir/bats-support/load.bash"
+load "$lib_dir/bats-assert/load.bash"
+load "$lib_dir/bats-file/load.bash"
+
+setup () {
+    test_temp_dir="$(temp_make --prefix 'otomi-values-')"
+    export ENV_DIR="$test_temp_dir"
+    env_path="$ENV_DIR/env"
+}
+
+teardown () {
+    temp_del "$test_temp_dir"
+    unset ENV_DIR
+    unset env_path
+}
+
+@test "the env folder should not exist without bootstrap.sh" { 
+    assert_file_not_exist "$env_path"
+}
```

Comment:
this is not testing anything, please remove

Reply:
instead you may add a test that checks if env files don't get overwritten when they exist

Reply:
Well I was watching some lectures on software testing and they mentioned a testrunner should at least have a case that 1) passes, 2) fails, and 3) passes after failing. So I think this would increase test coverage in the edge case covered by the test. I just wanted to deliver this feature so we can expand upon it in the future, so I didn't spend too much time overengineering.

Reply:
Ok, will have another look

Reply:
This statement could be part of `setup` function.
So before running any test you ensure that environment is clean.

Reply:
Yes that happens with line 10:

```
test_temp_dir="$(temp_make --prefix 'otomi-values-')"
```

Reply:
But I realise my mistake, I'm about to push some changes.